pkgs/development/tools/vagrant/default.nix

```
@@ -79,7 +79,8 @@ in buildRubyGem rec {
     in ''
     wrapProgram "$out/bin/vagrant" \
       --set GEM_PATH "${deps}/lib/ruby/gems/${ruby.version.libDir}" \
-      --prefix PATH ':' ${pathAdditions}
+      --prefix PATH ':' ${pathAdditions} \
+      --set VAGRANT_CHECKPOINT_DISABLE '✓'
```

Do we have to use these unicode?
```suggestion
      --set-default VAGRANT_CHECKPOINT_DISABLE 1
```

---

Thanks, I didn’t know about `--set-default`.

All non-empty values enable the disabling, so I avoided common representations of booleans (e.g. `true`, `yes`, `1`) so as not to imply that setting their opposite (`false`, `no`, `0`) would disable disabling. It’s confusing enough that this is a negative setting.

Happy to use whatever is conventional though.

---

Maybe there's no such convention here, just unicode sometimes breaks things in subtle ways, such as in https://github.com/NixOS/nixpkgs/pull/225965. And upstream is using `1` in their own tests: https://github.com/hashicorp/vagrant/blob/62c40e7395b57f32c84b95756e81b3164c17a131/test/unit/vagrant/util/checkpoint_client_test.rb#L37